### ISSUE TITLE
CvTabs content of subsequent tabs not hiding

### ISSUE THREAD

**Issue by gaabrielcs** [carbon-design-system/carbon-components-vue#664]
Testing the CvTabs both in the storybook and in my project, with the 2.17 release, results in the issue below.

<img width="659" alt="Screen Shot 2019-11-06 at 10 55 58" src="https://user-images.githubusercontent.com/50870887/68304536-b45e8d80-0084-11ea-9d91-2ae4df7710ea.png">

When I navigate to Tab 4 and get back to any of the previous it doesn't hide the content. 

Going through the code I saw this piece 
```
    onTabClick(id) {
      if (this.disabledTabs.indexOf(id) === -1) {
        if (this.selectedId !== id) {
          let newIndex = -1;
          this.selectedId = id;
          for (let i = 0; i < this.tabs.length && newIndex < 0; i++) {
            if (this.tabs[i].uid === id) {
              this.tabs[i].internalSelected = true;
              newIndex = i;
            } else {
              this.tabs[i].internalSelected = false;
            }
          }
          this.$emit('tab-selected', newIndex);
        }
      }
    },
```

In my opinion, debugging the logic, this `newIndex < 0` prevent any of the subsequent tabs to hide, after the one that was clicked.

**Comment by lee-chase** [comment]
Sorry yes, fix coming.

**Comment by lee-chase** [comment]
Fix published in 2.17.1 @gaabrielcs
